api/test/docker-deploy/docker-compose.yaml

```diff
@@ -18,7 +18,7 @@ version: "3.6"
 
 services:
   etcd:
-    image: quay.io/coreos/etcd:v3.4.0
+    image: gcr.io/etcd-development/etcd:v3.4.0
```

Comment:
Hi, I see there have some codes from other PRs 🤔

Reply:
ok ok will fix this right now